ga4gh/cli.py

```diff
@@ -1185,7 +1205,8 @@ def __init__(self, args):
         super(DestroyRunner, self).__init__(args)
 
     def run(self):
-        self.repoManager.destroy()
+        def func(): self.repoManager.destroy()
+        self.confirmRun(func, 'the Data Repository')
```

Comment:
Yes, but I'd rather keep the congruity with other confirmRun calls (plus, there's no guarantee that we won't add arguments to the destroy signature in the future)